docker/elasticsearch-setup/create-indices.sh

```diff
@@ -24,24 +24,24 @@ if [[ -z $ELASTICSEARCH_AUTH_HEADER ]]; then
 fi
 
 if [[ $ELASTICSEARCH_INSECURE ]]; then
-  ELASTICSEARCH_INSECURE="-k"
+  ELASTICSEARCH_INSECURE="-k "
 fi
 
 function create_datahub_usage_event_datastream() {
-  echo -e "Going to use prefix $INDEX_PREFIX"
   if [[ -z "$INDEX_PREFIX" ]]; then
     PREFIX=''
   else
     PREFIX="${INDEX_PREFIX}_"
   fi
-
-  POLICY_RESPONSE_CODE=$(curl -o /dev/null -s -w "%{http_code}" --header "$ELASTICSEARCH_AUTH_HEADER" "$ELASTICSEARCH_INSECURE" "$ELASTICSEARCH_PROTOCOL://$ELASTICSEARCH_HOST:$ELASTICSEARCH_PORT/_ilm/policy/${PREFIX}datahub_usage_event_policy")
+  echo -e "Create datahub_usage_event if needed against Elasticsearch at $ELASTICSEARCH_HOST:$ELASTICSEARCH_PORT" 
+  echo -e "Going to use index prefix:$PREFIX:"
+  POLICY_RESPONSE_CODE=$(curl -o /dev/null -s -w "%{http_code}" --header "$ELASTICSEARCH_AUTH_HEADER" "${ELASTICSEARCH_INSECURE}$ELASTICSEARCH_PROTOCOL://$ELASTICSEARCH_HOST:$ELASTICSEARCH_PORT/_ilm/policy/${PREFIX}datahub_usage_event_policy")
```

Comment:
Just curious -- Why is all of the new __STATUS stuff required?

Reply:
found the real problem... fixed up the PR